cmd/hatchet-migrate/migrate/migrations/20251007191452_v1_0_47.sql

```diff
@@ -0,0 +1,15 @@
+-- +goose Up
+-- +goose StatementBegin
+ALTER TYPE v1_payload_type ADD VALUE IF NOT EXISTS 'TASK_EVENT_DATA';
+ALTER TABLE v1_task_event
+    ADD COLUMN external_id UUID,
+    ADD COLUMN inserted_at TIMESTAMPTZ DEFAULT NOW();
```

Comment:
need these for storing payloads (the `inserted_at`, since `created_at` is a `TIMESTAMP(3)` and I didn't want to mess with changing the type, and `external_id` so we have a unique ID which we can use to look up the data by)